database/seeders/MismatchSeeder.php

```diff
@@ -20,8 +20,27 @@ public function run()
             ->for(User::factory()->uploader())
             ->create();
 
-        Mismatch::factory(42)
+        $expiredImport = ImportMeta::factory()
+            ->for(User::factory()->uploader())
+            ->expired()
+            ->create();
```

Comment:
We probably also want to make sure that the status of the expired import is 'completed'. Otherwise, we might get mismatches in the seeded database that come from pending of failed imports.

```suggestion
            ->create([
                'status' => 'completed'
            ]);
```